process_css_extend_template.xml

```diff
@@ -4,38 +4,34 @@
 --><project xmlns:dita="http://dita-ot.sourceforge.net" name="fox.jason.extend.css">
     
     <target name="extend.css.init">
+        <condition property="user.csspath" value="" else="${args.csspath}/">
```

Comment:
user.csspath belongs in extend.css.init with the other properties logically

`<condition>` can `@else` so there's no need to `<property>` user.csspath separately

Reply:
This shouldn't be the responsibility of this plugin anyway - removed